README.md

```diff
@@ -314,7 +314,7 @@ route:
 * `:idle-timeout`: optional; sets the maximum amount of time, measured in seconds, for the proxy to 
   wait for a response from the upstream server. If no response is received within the time
   specified, then an HTTP 504 error is returned. If this option is not specified then a value
-  of 60 seconds is used.
+  of 0 is used, which means that the connection does not ever timeout.
```

Comment:
Where did you get '0' from?  It looked like it might be 30 seconds from here - https://github.com/eclipse/jetty.project/blob/jetty-9.2.9.v20150224/jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java#L268-L271.